config/crd/bases/anywhere.eks.amazonaws.com_clusters.yaml

```diff
@@ -87,6 +87,11 @@ spec:
                     required:
                     - host
                     type: object
+                  labels:
```

Comment:
curious why move this

Reply:
I think this is the autogenerated CRD and it sorted the keys alphabetically.

Reply:
it's like auto generated from my side w `make release-manifest`

Reply:
ah yeah makes sense